src/vs/workbench/contrib/terminal/browser/remoteTerminalService.ts

```diff
@@ -36,7 +36,6 @@ export class RemoteTerminalService extends Disposable implements IRemoteTerminal
 	readonly onPtyHostRestart = this._onPtyHostRestart.event;
 
 	constructor(
-		@ITerminalInstanceService readonly terminalInstanceService: ITerminalInstanceService,
```

Comment:
this wasn't actually used anywhere and had to remove it bc otherwise would've been a cyclic dependency `terminalInstanceService -> remoteTerminalService -> terminalInstanceService`

Reply:
That works 👍

Reply:
I think remote/localterminalservice also don't end up depending on TerminalInstance which is good